### ISSUE TITLE
Prerequisite Question.

### ISSUE THREAD

**Issue by naoazure** [CROSS-Value-Chain/CVC-Mainnet-Release1#1]
@UzairLakhani123 
Is there any documentation for running DACs Node / Voting Node /J-Node?
If not, please specify how to set it up.

Are there any comments or documentation included with the additions you made from the original Geth?

**Comment by naoazure** [comment]
Thanks for the reply. Best regards.

You say that dts are included in the code, but are there any comments, etc. in the code?

Please also disclose how the different nodes are set up.
We would like to check the source this time as a project and be ready for anyone to rebuild.

What is the estimated completion date of the text for end users?

**Comment by dtsincpakistan** [comment]
Yes the source code contains comments. You can find the dts consensus mechanism inside the consensus package.

We have tested on our side that the uploaded code is building properly without any problem

Our tentative plan is to complete all items by the end of this month.
